13/alpine3.10/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN addgroup -g 1000 node \
       && case "${alpineArch##*-}" in \
         x86_64) \
           ARCH='x64' \
-          CHECKSUM="4c153345a08d2eeb40dfbb62f1ef6ade7ec369630f9cf9f061bf9d52b10acafc" \
+          CHECKSUM="" \
```

Comment:
Looks like this didn't finish yet

Reply:
Yeah I forgot to use "WIP" PR, I'll use PR title to indicate that.